application-xpoll-ui/src/main/resources/XPoll/XPollSheet.xml

```diff
@@ -522,7 +522,12 @@
   clear: both;
   visibility: hidden;
 }
-
+.clearfix {
+  display: flex;
+}
```

Comment:
This is a standard Bootstrap class. Changing its behavior doesn't look right. It's used in many places, so I'd be surprised if this change doesn't break anything.